src/app/workspace/resource/representation/document/document.component.ts

```diff
@@ -157,6 +166,7 @@ export class DocumentComponent implements OnInit, AfterViewInit {
 
         this._http.get(pathToJson, requestOptions).subscribe(
             res => {
+                console.log('res: ', res);
```

Comment:
Is this necessary to console log?

Reply:
definitely not 😅 removed in [1e2e4a3](https://github.com/dasch-swiss/dsp-app/pull/812/commits/1e2e4a31f6ae796f755abc88398d4821f752a38a)